apps/a11y-tests/package.json

```diff
@@ -16,7 +16,7 @@
     "axe-core": "3.5.0",
     "axe-puppeteer": "^1.0.0",
     "axe-sarif-converter": "^2.0.1",
-    "glob": "^7.1.2",
+    "glob": "7.1.6",
```

Comment:
Can you please restore version bumps in this and other files? It does not look intentional

Reply:
that was done by `syncpack:fix`... Not sure how to approach....

Reply:
> that was done by `syncpack:fix`...

Yeah, unfortunately sometimes it could do unexpected things... 😥

I propose to:
- revert version bumps across `package.json` (as I see only `packages/react-tabster/package.json` contains expected changes)
- and run `yarn` then